app/src/main/java/org/oppia/android/app/options/AudioLanguageFragment.kt

```diff
@@ -33,19 +42,23 @@ class AudioLanguageFragment : InjectableFragment(), AudioLanguageRadioButtonList
         savedInstanceState?.retrieveLanguageFromSavedState()
           ?: arguments?.retrieveLanguageFromArguments()
       ) { "Expected arguments to be passed to AudioLanguageFragment" }
-    return audioLanguageFragmentPresenter.handleOnCreateView(inflater, container, audioLanguage)
+    return if (enableOnboardingFlowV2.value) {
+      audioLanguageFragmentPresenter.handleCreateView(inflater, container)
+    } else {
+      audioLanguageFragmentPresenterV1.handleOnCreateView(inflater, container, audioLanguage)
+    }
   }
 
   override fun onSaveInstanceState(outState: Bundle) {
     super.onSaveInstanceState(outState)
     val state = AudioLanguageFragmentStateBundle.newBuilder().apply {
-      audioLanguage = audioLanguageFragmentPresenter.getLanguageSelected()
+      audioLanguage = audioLanguageFragmentPresenterV1.getLanguageSelected()
```

Comment:
For this & below: are these valid interactions with ``audioLanguageFragmentPresenterV1`` if the parameter is enabled (i.e. that the new one is being used)?

Reply:
No, thanks for catching this. I have gated them behind the platform parameter.